.github/CODEOWNERS

```diff
@@ -7,6 +7,57 @@
 # Modification to what is considered "core packages" require admin approval.
 /SPECS/core-packages/* @microsoft/cbl-mariner-admins
 
+# Modification to specific packages go to specific teams
+/SPECS/installkernel/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-azure/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-hci/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-headers/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-mshv/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-uvm/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-hci-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-azure-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-mstflint-signed/* @microsoft/cbl-mariner-kernel
+
+/SPECS/grub2/* @microsoft/cbl-mariner-bootloader
+/SPECS/grubby/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned-x64/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned-aarch64/* @microsoft/cbl-mariner-bootloader
+/SPECS-SIGNED/grub2-efi-binary-signed/* @microsoft/cbl-mariner-bootloader
+
+/SPECS/dracut/* @microsoft/cbl-mariner-dracut
+/SPECS/initramfs/* @microsoft/cbl-mariner-dracut
+
+/SPECS/systemd/* @microsoft/cbl-mariner-systemd
+
+/SPECS/bcc/* @microsoft/cbl-mariner-debug-tools
+/SPECS/bpftrace/* @microsoft/cbl-mariner-debug-tools
+/SPECS/crash/* @microsoft/cbl-mariner-debug-tools
+/SPECS/gdb/* @microsoft/cbl-mariner-debug-tools
+/SPECS/kexec-tools/* @microsoft/cbl-mariner-debug-tools
+
+/SPECS/openssl/* @microsoft/cbl-mariner-openssl
+
+/SPECS/dnf/* @microsoft/cbl-mariner-package-managers
+/SPECS/dnf-plugins-core/* @microsoft/cbl-mariner-package-managers
+/SPECS/rpm/* @microsoft/cbl-mariner-package-managers
+/SPECS/tdnf/* @microsoft/cbl-mariner-package-managers
+
+/SPECS/moby-buildx/* @microsoft/cbl-mariner-container-runtime
+/SPECS/moby-cli/* @microsoft/cbl-mariner-container-runtime
+/SPECS/moby-containerd/* @microsoft/cbl-mariner-container-runtime
+/SPECS/moby-engine/* @microsoft/cbl-mariner-container-runtime
+/SPECS/moby-runc/* @microsoft/cbl-mariner-container-runtime
+/SPECS/kata-containers/* @microsoft/cbl-mariner-container-runtime
```

Comment:
We may want to have kata-containers-cc and moby-containerd-cc here as well

Reply:
Added